docs/learning/quantity/index.md

```diff
@@ -219,3 +150,117 @@ Each Format defined in a FormatSet need to be mapped to a valid [ECName](../../b
 ```
 
 </details>
+
+## Examples of Usage
+
+### Numeric Format
+
+  The example below uses a simple numeric format and generates a formatted string with 4 decimal place precision. For numeric formats there is no conversion to other units; the unit passed in is the unit returned with the unit label appended if "showUnitLabel" trait is set.
+<details>
+<summary>Example Code</summary>
+
+```ts
+[[include:Quantity_Formatting.Numeric]]
+```
+
+</details>
+
+### Composite Format
+
+For the composite format below, we provide a unit in meters and produce a formatted string showing feet and inches to a precision of 1/8th inch.
+
+<details>
+<summary>Example Code</summary>
+
+```ts
+[[include:Quantity_Formatting.Composite]]
+```
+
+</details>
+
+### Parsing Values
+
+<details>
+  <summary>Example Code:</summary>
+
+```ts
+[[include:Quantity_Formatting.Simple_Parsing]]
+```
+
+</details>
+
+### Using a FormatsProvider
+
+The example below uses the SchemaFormatsProvider, an implementation of a FormatsProvider, found in `ecschema-metadata` to format values associated with the length of an object.
```

Comment:
APIs should be referenced appropriately (below as well)

Reply:
I've wrapped APIs in backticks